packages/pg/lib/utils.js

```diff
@@ -23,8 +23,17 @@ function arrayString(val) {
       result = result + 'NULL'
     } else if (Array.isArray(val[i])) {
       result = result + arrayString(val[i])
-    } else if (val[i] instanceof Buffer) {
-      result += '\\\\x' + val[i].toString('hex')
+    } else if (ArrayBuffer.isView(val[i])) {
+      var item = val[i]
+      if (!(item instanceof Buffer)) {
```

Comment:
```suggestion
      if (!Buffer.isBuffer(item)) {
```

Reply:
Was trying to follow the style used elsewhere in this file (see above). Happy to change if we're otherwise supportive of this direction.